tdrs-backend/scripts/README.md

```diff
@@ -0,0 +1,64 @@
+# Database backup
+
+The backup process in cloudfoundry is done using the pg_dump utility from Postgres client 
+version 12. According to [Cloud.Gov documentation](https://cloud.gov/docs/services/relational-database/#backups) 
+the RDS service that is used by Cloud.Gov uses 14 days retention for automatic backups. These backups can be restored by 
+Cloud.Gov support.
+
+There are also other approaches to access database data by performing backups using 
+pg_dump such as using: 
+```bash
+cg-manage-rds export -o "-F c" -f ./backup.pg ${SERVICE_NAME}
+```
+
+This command uses "pg_dump -O" to create backup, which doesn't set the ownership on the backup 
+file.
+Another way to create backup is to use cf-service-connect and to run pg_dump command
+(see [Cloud.Gov documentation](https://cloud.gov/docs/services/relational-database/#using-cf-service-connect-plugin)). 
+
+However, the caveat is that we have to first connect to the database service and keep that 
+connection live, while performing backup using cloudfoundry connection. This causes issues 
+while we need to create shell scripts.
+
+The db_backup.py is written as a python script to perform backup and restore commands.
+
++ [list](./README.md#lkjlk)
+
++ Backup database 
++ Restore the backup 
++ Save local file to S3 instance 
++ Download remote s3 file 
++ List S3 files (To be added)
+
+### Preparation: 
+We need to have postgres12 and Python available before being able to use the script: 
+```bash
+export LD_LIBRARY_PATH=~/deps/#/python/lib
+```
+Note: '#' can change depending on programs/packages installed on the instance. Normally # == 0 or 1
+
+Then:
+```bash
+.~/deps/#/python/python3
```

Comment:
Is the '.' a typo? This seems to fail.

Reply:
Also, looks like it's missing 'bin/' between the two pythons

Reply:
Nice catch! /bin was missed.